modelina-website/src/components/InfoModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+export default function InfoModal(props: { text: string; html: any; }){
+
+    const [showModal, setShowModal] = useState(false);
+
+    const onOpenModal = () => {
+        setShowModal(true)
+        document.body.style.overflowY = 'hidden'
+    };
+
+    const onCloseModal = () => {
+        setShowModal(false)
+        document.body.style.overflowY = 'scroll'
+    };
+
+    const MyModal = () => {
+        return (
+            <>  
+                <div className='fixed inset-0 bg-black bg-opacity-10' onClick={onCloseModal}></div>
```

Comment:
is it possible we can make the model width larger on smaller screens? 🤔

Reply:
@jonaslagoni  ya sure , it could be done by changing the value of `min-w-[]` in `<div className="max-h-32 overflow-y-auto min-w-40">`

Reply:
in this article of [tailwind responsive design](https://tailwindcss.com/docs/responsive-design) it is recommended to keep width somewhere between 32 to 48 for medium to large devices so i made the decision to keep it 40, if you want it to be a specific value plz let me know. :smile_cat: i

Reply:
Alright, lets keep it then 👍